### DISCUSSION TITLE
v1.1.0

### DISCUSSION THREAD

**Discussion by Mrs4s** [Mrs4s/go-cqhttp#2241] [General]
## Changelog
此版本支持了自定义 `SignServer` 功能, 配置后可解决 `code: 45` 错误和部分消息发送的风控.

## 新增:
9c1390c 支持了 `SignServer` 的配置, @Mrs4s

## 修复
修改网易云音乐url格式. @Azusa-Yuan in https://github.com/Mrs4s/go-cqhttp/pull/2146
修复上传文件后没有解除占用的问题.  @Janet-Baker in https://github.com/Mrs4s/go-cqhttp/pull/2186
修复群匿名消息事件中的重复的sub_type #2216 by @xiangxiangxiong9 in https://github.com/Mrs4s/go-cqhttp/pull/2219

<hr /><em>This discussion was created from the release <a href='https://github.com/Mrs4s/go-cqhttp/releases/tag/v1.1.0'>v1.1.0</a>.</em>

**Comment by HRxuan** [comment]
At present, the mobile phone is configured to start go-cqhttp, and timeout access will occur.
